templates/testimonies/testimonies.html.twig

```diff
@@ -20,23 +20,27 @@
             <img class="d-none d-sm-inline-block"
                  src="{{ asset('build/images/doublequotes3.png') }}" alt="quotes">
         </h2>
-        <div class="container">
-            <div class="row mx-0">
-                {% for testimony in testimonies %}
-                    <div class="col-md-6 col-lg-4 d-flex justify-content-center">
-                        <div class="card mb-5 text-center">
-                            <img src="{{ asset('uploads/logos/' ~ testimony.ecosystem.logo) }}" class="card-img-top img-fluid" alt="logo">
+
+        <div class="container testimonies-container">
+            {% for testimony in testimonies %}
+                <div class="card border-0">
+                    <div class="row mx-0 no-gutters
+                        {% if loop.index % 2 == 0 %}
```

Comment:
ça ça devrait fonctionner (sur une ligne)
{{ loop.index is even ? 'flex-row-reverse'  }}